websocket/frontend/src/todolist/TodoList.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import TodoForm from "./Todoform"
+import TodoItem from "./TodoItem"
+
+const api_url = 'http://localhost:3001/api/v1/todos/'
```

Comment:
1. Why is `api_url` not in camel case which is the standard for js?
2. Remove domain name from the URL. Else, when this gets deployed, it will fail